backend/src/v5/utils/sso/aad.js

```diff
@@ -0,0 +1,40 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const config = require('../config');
+const msal = require('@azure/msal-node');
+
+const Aad = {};
+
+let clientApplication;
+
+Aad.getClientApplication = () => {
+	const clientAppConfig = { auth: config.sso.aad };
```

Comment:
could we also configure the logging to log into our logger (with the right labelling)